diff -Naur a/networking/ifupdown.c b/networking/ifupdown.c
--- a/networking/ifupdown.c	2015-01-27 09:48:58.000000000 +0100
+++ b/networking/ifupdown.c	2016-06-03 10:19:47.144427376 +0200
@@ -526,7 +526,7 @@
 		"dhcpcd -k %iface%",
 	},
 	{ "dhclient",
-		"dhclient -pf /var/run/dhclient.%iface%.pid %iface%",
+		"dhclient -nw -pf /var/run/dhclient.%iface%.pid %iface%",
 		"kill -9 `cat /var/run/dhclient.%iface%.pid` 2>/dev/null",
 	},
 	{ "pump",
