From b8e76c17ecf1dcf726cdd37146a1132adba986bb Mon Sep 17 00:00:00 2001
From: Alexandre Abadie <alexandre.abadie@inria.fr>
Date: Mon, 22 Oct 2018 13:53:14 +0200
Subject: [PATCH] fix makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2f45e31..be3658e 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ OBJS = $(addprefix ykushcmd/objs/, ykushcmd.o commandParser.o ykushxs.o yk_usb_d
 LIBS = -lhidapi-libusb -lusb-1.0 -ludev
 LOADPATHS = -L$(CUR_PATH)/ykushcmd/linux
 INCLUDEPATHS= -I$(CUR_PATH)/ykushcmd/linux -I$(CUR_PATH)/ykushcmd/ykushxs -I$(CUR_PATH)/ykushcmd/ykush -I$(CUR_PATH)/ykushcmd -I$(CUR_PATH)/ykushcmd/help -I$(CUR_PATH)/ykushcmd/ykush2 -I$(CUR_PATH)/ykushcmd/ykush3
-CPP = g++
+CPP = $(CXX)
 
 ykushcmd : $(OBJS)
 	$(CPP) $(LOADPATHS) -o bin/ykushcmd $(OBJS) $(LIBS)
-- 
2.17.1

