diff --git a/lora_pkt_fwd/src/lora_pkt_fwd.c b/lora_pkt_fwd/src/lora_pkt_fwd.c
index 31a3743..c15b191 100644
--- a/lora_pkt_fwd/src/lora_pkt_fwd.c
+++ b/lora_pkt_fwd/src/lora_pkt_fwd.c
@@ -2143,6 +2143,12 @@ void thread_down(void) {
                 txpkt.no_crc = (bool)json_value_get_boolean(val);
             }
 
+            /* Parse "No Header" flag (optional field) */
+            val = json_object_get_value(txpk_obj,"nhdr");
+            if (val != NULL) {
+                txpkt.no_header = (bool)json_value_get_boolean(val);
+            }
+
             /* parse target frequency (mandatory) */
             val = json_object_get_value(txpk_obj,"freq");
             if (val == NULL) {
